### ISSUE TITLE
Moving PRs to directory structure

### ISSUE THREAD

**Issue by apayne97** [choderalab/asapdiscovery#143]
# Description
It looks like merging existing PRs into main will be a bit tricky now that we've restructured the whole directory.
If I understand it correctly, if you just merge main into the branch, instead of just adding the new directory structure, it also deletes any files that didn't exist in here #98 before it was merged into main.

I've found that this method works:
1. In your branch, for any files you would like to see get pulled into main, before merging main into the branch, create the new directory structure and move your new files there.
2. Then merge from main
3. Then delete any old directories (? not sure if this is necessary but they do seem to linger, especially if you have any uncommitted files in them)

# TODO
Here are all the existing PRs that will need this done.
- [ ] #92 
- [ ] #104 
- [ ] #105 
- [ ] #113 
- [ ] #117 
- [ ] #128 
- [ ] #129 
- [ ] #130 
- [ ] #135 
- [ ] #141 
- [ ] #142 

# Questions
- [ ] is there a better way to do this?

**Comment by dotsdl** [comment]
I'm working on #117, and just did the following:

```bash
git checkout add-postera-api

# make a new branch to try the merge on; easy to abandon if messy
git checkout -b add-postera-api-merge
git merge main
```

After doing this, the files committed under `asapdiscovery` remain, and the new separate packages, e.g. `asapdiscovery-data`,  are introduced as well. This was a simple PR with completely separate files to those already in existence in the repo, so may not be representative of others.

@apayne97 when you do a merge of `main` into a PR that changes files that were moved into the new structure, are your changes present in the moved versions of those files? I suspect they should be, and files that weren't present during the restructuring are left behind at their existing paths.

**Comment by ijpulidos** [comment]
@dotsdl Yes, that's basically the approach I used in #135 , I did have to resolve some merge conflicts though, but they were easily solvable.

**Comment by apayne97** [comment]
I have now done this for #104 (the PR that keeps me up at night)! this seems to work!

**Comment by apayne97** [comment]
Completed!
